espRFID/UpdateACLTask.cpp

```diff
@@ -36,7 +36,9 @@ const char* aclFilename = "acl";
 
 
 UpdateACLTask::UpdateACLTask(uint32_t interval, DoorLatchTask& _latchTask, BlinkPatternTask& _blinkTask)
-: TimedTask(0)
+: TimedTask(10000)
```

Comment:
Instead of a blind 10 second sleep, it could wait until a connection is made.  The ServerTask waits for a connection, it could call a method on this class to start the automatic updates when connection has been made.